docs/integration-testing.md

```diff
@@ -0,0 +1,86 @@
+# Integration Testing
+
+Integration testing is the phase in software testing in which individual software modules are combined and tested as a group. 
+Integration testing is conducted to evaluate the compliance of a system or component with specified functional requirements. 
+It occurs after unit testing and before system testing. Integration testing takes as its input modules that have been unit tested, groups them in larger aggregates, applies tests defined in an integration test plan to those aggregates, and delivers as its output the integrated system ready for system testing.
+
+To group these integration test aggregates, we make use of test suites. In software development, a test suite, less commonly known as a validation suite, is a collection of test cases that are intended to be used to test a software program to show that it has some specified set of behaviours. 
+A test suite often contains detailed instructions or goals for each collection of test cases and information on the system configuration to be used during testing. 
```

Comment:
That's what wiki says and rather vague. Do we want to give more specific guidance for Kyma purposes? 

```suggestion
For each test suite, you should provide the following information:
- Detailed instructions and goals
- The system configuration to be used during testing
- Prerequisite states or step
- Information about the following tests 
```